pkg/defaultcomponents/defaults.go

```diff
@@ -158,6 +159,7 @@ func Components() (otelcol.Factories, error) {
 		prometheusexporter.NewFactory(),
 		fileexporter.NewFactory(),
 		kafkaexporter.NewFactory(),
+		debugexporter.NewFactory(),
 		loggingexporter.NewFactory(),
```

Comment:
should loggingexporter.NewFactory() be removed?

Reply:
No, we want to keep that around so that existing configurations do not break.  We could introduce a config processor that would translate the configuration from one to the other, but that's a next step.